### ISSUE TITLE
Flutter App Warning: "The app has exposed Google Cloud Platform (GCP) API keys"

### ISSUE THREAD

**Issue by gabrielgpavan** [OneSignal/OneSignal-Flutter-SDK#226]
I am having the following error, could someone explain to me why? And how can I solve it?
Flutter program (version 1.12.13 hotfix 9)

```
The app has exposed Google Cloud Platform (GCP) API keys. Read this Google Help Center article for more details.

Vulnerable places:

com.onesignal.h2-> d
Affects APK versions 20 and 19.
```

**Comment by rymesaint** [comment]
yeah i have the same problem too.

**Comment by aleksandar-radivojevic** [comment]
Same here, my app is still pending publication, so is there going to be a problem with publishing my app because of this error? Please help.

Flutter framework(1.12.13+ hotfix.8)
onesignal_flutter package (2.3.4 version in pubspec.yaml and 3.12.7 in build.gradle).
The error that I'm getting from Play console: 

Security alert
Your app contains exposed Google Cloud Platform (GCP) API keys. Please see this Google Help Centre article for details.
Vulnerable locations:
com.onesignal.mc->d
Affects APK version 1.

**Comment by AlvaroVasconcelos** [comment]
yeah i have the same problem too.

**Comment by jkasten2** [comment]
This has been fixed in the 2.4.0 release
https://github.com/OneSignal/OneSignal-Flutter-SDK/releases/tag/2.4.0

**Comment by Aina27** [comment]
Same here, i'm using onesignal_flutter for Flutter, I already updated to the lastest SDK version 3.1.3 
 I still got this on Google Play Console:

Your app contains exposed Google Cloud Platform (GCP) API keys. Please see this Google Help Center article for more information. 
Vulnerable sections:
com.onesignal.lc-> d
com.onesignal.mc-> d
Applies to versions 34010, 32014 and 32010 of the APK file, as well as other versions.
